pyproject.toml

```
@@ -34,7 +34,7 @@ pyvisa-py = { version = "0.5.3", optional = true }
 qm-qua = { version = "==1.1.1", optional = true }
 qualang-tools = { version = "==0.14.0", optional = true}
 setuptools = { version = ">67.0.0", optional = true }
-laboneq = { version = ">=2.12.0", optional = true }
+laboneq = { version = ">=2.20.0", optional = true }
```

We might also consider fixing the version
```suggestion
laboneq = { version = "==2.20.0", optional = true }
```
as discussed in #642